### PR TITLE
PLNSRVCE-688: Add jsweet maven repository for org.jsweet.ext:typescri…

### PR DESCRIPTION
**Pull request by dwalluck**
…pt.java-ts.core:jar:1.4.0.1

Required by apicurio-data-models-1.1.26.

### CODE REVIEW

**Comment by dwalluck** [comment]
I never updated the `BUILD_POLICY_DEFAULT_STORE_LIST` value for an repositories I added. I can do that in a separate PR.

File: deploy/cache/base/deployment.yaml [hunk 1]
```diff
@@ -51,6 +51,10 @@ spec:
               value: https://maven.repository.redhat.com/ga
             - name: STORE_REDHAT_TYPE
               value: maven2
+            - name: STORE_JSWEET_URL
```
Comment (stuartwdouglas) [inline deploy/cache/base/deployment.yaml:54]: This file has no effect any more, it was still around for the transtion  but has just been deleted.

**Review by stuartwdouglas — APPROVED** [review]
LGTM  once the conflict is fixed.

File: deploy/cache/base/deployment.yaml (hunk 1 above)
Comment (dwalluck) [reply deploy/cache/base/deployment.yaml]: Removed from this PR.

**Comment by stuartwdouglas** [comment]
Note that BUILD_POLICY_DEFAULT_STORE_LIST is not really needed any more, this is automatically populated by the operator when it sets up the cache.

**Pull request merged**